src/test/java/org/folio/rest/api/LoansApiTest.java

```diff
@@ -178,6 +178,212 @@ public void canCreateALoan()
   }
 
   @Test
+  public void canCreateALoanForDcb()
+    throws MalformedURLException,
+    InterruptedException,
+    ExecutionException,
+    TimeoutException {
+
+    UUID id = UUID.randomUUID();
+    UUID itemId = UUID.randomUUID();
+    UUID itemLocationAtCheckOut = UUID.randomUUID();
+    UUID userId = UUID.randomUUID();
+    UUID proxyUserId = UUID.randomUUID();
+    UUID loanPolicyId = UUID.randomUUID();
+    DateTime expectedLostDate = DateTime.now();
+    UUID overdueFinePolicyId = UUID.randomUUID();
+    UUID lostItemPolicyId = UUID.randomUUID();
+    final DateTime claimedReturnedDate = DateTime.now(DateTimeZone.UTC);
+    final DateTime agedToLostDate = DateTime.now(DateTimeZone.UTC).minusDays(10);
+    DateTime dateLostItemShouldBeBilled = new DateTime(2017, 9, 27, 10, 23, 43, DateTimeZone.UTC);
+
+    JsonObject loanRequest = new LoanRequestBuilder()
+      .withId(id)
+      .withItemId(itemId)
+      .withUserId(userId)
+      .withProxyUserId(proxyUserId)
+      .withLoanDate(new DateTime(2017, 6, 27, 10, 23, 43, DateTimeZone.UTC))
+      .open()
+      .withAction("checkedout")
+      .withItemStatus("Checked out")
+      .withDueDate(new DateTime(2017, 7, 27, 10, 23, 43, DateTimeZone.UTC))
+      .withItemEffectiveLocationIdAtCheckOut(itemLocationAtCheckOut)
+      .withLoanPolicyId(loanPolicyId)
+      .withDeclaredLostDate(expectedLostDate)
+      .withOverdueFinePolicyId(overdueFinePolicyId)
+      .withLostItemPolicyId(lostItemPolicyId)
+      .withClaimedReturnedDate(claimedReturnedDate)
+      .withAgedToLostDelayedBilling(false, dateLostItemShouldBeBilled, agedToLostDate)
+      .withIsDcb(true)
+      .create();
+
+    JsonObject loan = loansClient.create(loanRequest).getJson();
+
+    assertThat("id does not match",
```

Comment:
Do we really have to check every property of the loan in these test cases? We are only adding one, so maybe checking that this new field can be read and written is enough?